sonar-batch/src/main/java/org/sonar/batch/qualitygate/QualityGateVerifier.java

```diff
@@ -160,10 +166,11 @@ private String getAlertLabel(Resource project, ResolvedCondition condition, Metr
       .append(" ").append(operatorLabel(condition.operator())).append(" ")
       .append(alertValue(condition, level));
 
-    if (alertPeriod != null) {
-      Snapshot snapshot = resourceCache.get(project).snapshot();
-      stringBuilder.append(" ").append(periods.label(snapshot, alertPeriod));
-    }
+    // Disabled because snapshot is no more created by the batch
```

Comment:
Because it will be removed when merging on master as you have moved this to the CE !